Atomic/syscontainers.py

```diff
@@ -1297,6 +1301,7 @@ def rollback(self, name):
         installed_files_template = info["installed-files-template"] if "installed-files-template" in info and rpm_installed is None else None
         has_container_service = info["has-container-service"] if "has-container-service" in info else True
         rename_files = info["rename-installed-files"] if "rename-installed-files" in info else None
+        use_links = info["use-links"] if "use-links" in info else False
```

Comment:
> Specify if files copied to the host under */usr* should use
+hard links when possible.  By default it is True.

Shouldn't we default to True here?